Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java

```diff
@@ -718,4 +718,18 @@ public void setFilter(String filter) {
         super.filter = filter;
         this.lazyDTOModel.setFilterString(filter);
     }
+
+    /**
+     * Check and return whether LDAP group and LDAP server are configured for current user
+     * and if LDAP server is read only.
+     *
+     * @return whether LDAP server is configured and read only
+     */
+    public boolean isLdapServerReadOnly() {
+        try {
+            return this.userObject.getLdapGroup().getLdapServer().isReadOnly();
+        } catch (NullPointerException e) {
```

Comment:
Please use null checks instead of Exception Handling!

Reply:
This is the rare case where I opted for a more "phytonic" way of error handling ("asking for forgivness is easier than asking for permission"), because it seemed more appropriate and concise in this case. Still not acceptable?

Reply:
I had overlooked that, but yeah, that's not how it should be. In particular, we know the case, it's well-defined: If a user doesn't have an LDAP group, then it's a database user. An LDAP group without an LDAP server should actually no longer be possible (```CONSTRAINT `FK_ldapGroup_ldapServer_id` FOREIGN KEY (`ldapserver_id`) REFERENCES `ldapserver` (`id`)```). I don't know if the case _should_ be caught at all.